presto-main-base/src/test/java/com/facebook/presto/sql/planner/TestAccessControlFiltersMasks.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.planner;
+
+import com.facebook.presto.common.QualifiedObjectName;
+import com.facebook.presto.spi.plan.FilterNode;
+import com.facebook.presto.spi.plan.SemiJoinNode;
+import com.facebook.presto.spi.plan.TableScanNode;
+import com.facebook.presto.spi.security.ViewExpression;
+import com.facebook.presto.sql.planner.assertions.BasePlanTest;
+import com.facebook.presto.testing.LocalQueryRunner;
+import com.facebook.presto.testing.TestingAccessControlManager;
+import com.google.common.collect.ImmutableMap;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.Optional;
+
+import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.anyTree;
+import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.expression;
+import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.node;
+import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.project;
+
+public class TestAccessControlFiltersMasks
+        extends BasePlanTest
+{
+    private static final String CATALOG = "local";
+    private static final String USER = "user";
+    private static final String RUN_AS_USER = "run-as-user";
+
+    private LocalQueryRunner runner;
+    private TestingAccessControlManager accessControl;
+
+    @BeforeClass
+    public void init()
+    {
+        runner = getQueryRunner();
+        accessControl = getQueryRunner().getAccessControl();
+    }
+
+    @Test
+    public void testBasicRowFilter()
+    {
+        executeExclusively(() -> {
+            accessControl.rowFilter(
+                    new QualifiedObjectName(CATALOG, "tiny", "orders"),
+                    USER,
+                    new ViewExpression(USER, Optional.empty(), Optional.empty(), "orderkey < 10"));
+            assertPlan("SELECT * FROM orders",
+                    anyTree(
+                            node(FilterNode.class, node(TableScanNode.class))));
```

Comment:
Can you add more a more detailed filter matcher ? [an example](https://github.com/prestodb/presto/blob/47a708022aba003ebfda483ea3c940231fdbd282/presto-main-base/src/test/java/com/facebook/presto/sql/planner/TestLogicalPlanner.java#L421)

Reply:
done